build.gradle

```diff
@@ -150,6 +151,10 @@ subprojects { subproject ->
 			append = false
 			destinationFile = file("$buildDir/jacoco.exec")
 		}
+		if (isTravisBuild) {
+			// restrict memory usage, so tests don't fail with exit code 137 on travis
+			maxHeapSize = '256m'
```

Comment:
@panchenko ,

Looks like now we adjust only `Xmx` .
Isn't that Travis option really enough:
```
env:
  global:
  - MALLOC_ARENA_MAX=1
  - GRADLE_OPTS="-Xmx512m"
```
?